recipes-bsp/u-boot/u-boot_2019.04.bbappend

```diff
@@ -18,4 +18,36 @@ SRC_URI += "file://0001-riscv-add-infrastructure-for-calling-functions-on-ot.pat
             file://0016-riscv-dts-ae350-support-SMP.patch \
             file://0017-riscv-ae350-enable-SMP.patch \
             file://0018-riscv-dts-fix-CONFIG_DEFAULT_DEVICE_TREE-failure.patch \
+            file://0001-sifive-fu540-Set-default-arguments-to-help-booting.patch \
+            file://tftp-boot.txt \
            "
+
+DEPENDS += "u-boot-tools-native"
+
+UBOOT_ENTRYPOINT = "0x80200000"
+
+# Overwrite this for your server
+TFTP_SERVER_IP ?= "127.0.0.1"
+
+u_boot_do_patch() {
+      sed -i -e 's,@SERVERIP@,${TFTP_SERVER_IP},g' ${WORKDIR}/0001-sifive-fu540-Set-default-arguments-to-help-booting.patch
```

Comment:
Can we do it after patching ? say via do_configure_prepend, editing patches seems quite cryptic to me.

Reply:
I did think of that, but that seems just as messy. If you prefer that I'll give it a shot.

Reply:
hmm how many such instances are there ?

Reply:
Just 1, it actually seems easy. Just testing it now